interpreter/shared/src/main/scala/org/ergoplatform/validation/SigmaValidationSettingsSerializer.scala

```diff
@@ -27,10 +28,12 @@ object SigmaValidationSettingsSerializer extends SigmaSerializer[SigmaValidation
       val status = RuleStatusSerializer.parse(r)
       ruleId -> status
     }
-    val initVs = ValidationRules.currentSettings
-    val res = parsed
-      .filter(pair => initVs.get(pair._1).isDefined)
-      .foldLeft(initVs) { (vs, rule) => vs.updated(rule._1, rule._2) }
+    val map = parsed.map { case (id, status) =>
+      val (rule, _) = ValidationRules.currentSettings.get(id)
+          .getOrElse(throw SerializerException(s"Rule with id $id is not registered"))
```

Comment:
Looks like consensus-breaking change: before, settings not in the currentSettings were ignored just, now they are failing deserialization

Reply:
This serialiser is only use in JsonCodecs to serialise Context data. Not part of the consensus.